packages/utilities/src/focus.ts

```diff
@@ -99,8 +99,25 @@ export function getPreviousElement(
       allowFocusRoot,
       tabbable);
 
-    if (childMatch && ((tabbable && (isElementTabbable(childMatch, true))) || !tabbable)) {
-      return childMatch;
+    if (childMatch) {
```

Comment:
Is it possible to write a unit test for this scenario?

Also, is there logic missing on "findNextElement"?

Reply:
getNextElement doesn't need to be updated, the logic is slightly different and doesn't check for tabindex when checking if an element is tabbable. There may be some other bug hidden in here from that difference, but it doesn't need this update

Reply:
and yes, I'll look at adding a unit test

Reply:
cool ping me once you have a test to cover it (so we don't break it later!) and let's get it merged.

Reply:
@dzearing done, ut added. I also verified that this breaks without my fix and works with it :)